wolfcrypt/src/ecc.c

```diff
@@ -3828,6 +3828,7 @@ int wc_ecc_mulmod_ex2(const mp_int* k, ecc_point* G, ecc_point* R, mp_int* a,
 #endif
    /* k can't have more bits than order */
    if (mp_count_bits(k) > mp_count_bits(order)) {
+      WOLFSSL_MSG("WARNING: mp_count_bits(k) > mp_count_bits(order)");
```

Comment:
"Private key length is greater than order in bits."

Reply:
Good suggestion. Done